letsencrypt/storage.py

```diff
@@ -0,0 +1,427 @@
+"""The RenewableCert class, representing renewable lineages of
+certificates and storing the associated cert data and metadata."""
+
+import configobj
+import copy
+import datetime
+import os
+import OpenSSL
+import parsedatetime
+import pyrfc3339
+import pytz
+import re
+import time
+
+from letsencrypt import le_util
+
+DEFAULTS = configobj.ConfigObj("renewal.conf")
```

Comment:
`DEFAULTS` has already been defined in `renewal.py`... However, please first see my comments at https://github.com/letsencrypt/lets-encrypt-preview/pull/404/files#r30349259...